src/physics.js

```diff
@@ -29,46 +35,66 @@ export default class Physics {
         physicsBody.velocity.add(Victor(0, (physicsBody.accel.y - this.gravity) * this.timeStepFactor));
 
         if (physicsBody.velocity.y !== 0) {
-            this.quadTree.remove(physicsBody, 'quadID');
+            this.quadTree.remove(physicsBody, 'currentID');
             physicsBody.aabb.add(Victor(0, physicsBody.velocity.y * this.timeStepFactor));
             this.quadTree.put(physicsBody);
 
-            this.quadTree.get(physicsBody, (nearbyBody) => {
-                if (nearbyBody.quadID !== physicsBody.quadID && this.collides(physicsBody, nearbyBody)) {
-                    physicsBody.aabb.subtract(Victor(0, physicsBody.velocity.y * this.timeStepFactor));
-                    physicsBody.velocity.y = 0;
+            //const nearArr = this.quadTree.get(physicsBody);
+            //console.log(`length: ${nearArr.length}`);
 
-                    return false;
-                } else {
-                    return true;
+            let bool = true;
```

Comment:
Suggesting changing `bool` to `firstCollision` or something that's meaningful?  Bool doesn't mean anything to me.  That's like naming `firstName` as `string`